handlers/command_handler.py

```diff
@@ -113,6 +115,7 @@ def _delete_first_two_words(self, phrase: str) -> str:
         await self.client.send_message(message.channel, reason)
 
     async def command_status(self, message):
+        mention_user = "{0.author.mention}".format(message)
```

Comment:
When defining a string with a variable, you can just put the `message.author.mention` part instead of having to format the string

Also, please instead do `<@{id}>` (see above)

Reply:
So instead of the code being:
```
mention_user = "{0.author.mention}".format(message)
```
it should change to:
```
mention_user = "<@{0.author.id}>"
```

Reply:
you will need `"<@{0.author.id}>".format(message)` since you have other characters in the string